src/components/TeamMemberTasks/TeamMemberTasks.jsx

```diff
@@ -226,10 +230,31 @@ const TeamMemberTasks = props => {
                     };
                   }
                 }
+                
+//Add time entry property to task
+                let allTimeEntries = []
+                finalData.forEach((member)=>{
+                  member.tasks.forEach((task)=>{
+                  task.allTimeEntries = allTimeEntries
+                  member.timeEntries.forEach((entry)=>{
+                    if (task.projectId === entry.projectId ){
+                   task.allTimeEntries.push(entry)
+                  
+                    }else{
```

Comment:
The code format here looks a bit weird, did you have Prettier installed @IamDavidOkeke 
<img width="681" alt="image" src="https://user-images.githubusercontent.com/5071040/168685306-ee46559d-1ccf-4aaa-84df-f08d9bfa063b.png">